src/foam/u2/tag/Image.js

```diff
@@ -30,18 +66,50 @@ foam.CLASS({
       name: 'displayHeight',
       attribute: true
     },
-    ['alpha', 1.0]
+    ['alpha', 1.0], 
+    ['role', 'presentation']
```

Comment:
This may not be the case all the time, we would be setting the default to something that is not CSS default.